components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/saml/SAML2BearerGrantHandlerTest.java

```diff
@@ -485,4 +491,24 @@ public void handleSAML2Token(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
         when(oAuthServerConfiguration.getSAML2TokenCallbackHandler()).thenReturn(callbackHandler);
     }
 
+    private void prepareForGetSAMLSSOServiceProvider() throws Exception {
```

Comment:
Add new line after method signature

Reply:
Addressed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1438